src/board/nap/nap_common.c

```diff
@@ -171,16 +171,21 @@ void nap_rd_dna_callback(u16 sender_id, u8 len, u8 msg[], void* context)
   (void)sender_id; (void)len; (void)msg; (void) context;
   u8 dna[8];
   nap_rd_dna(dna);
-  sbp_send_msg(SBP_MSG_NAP_DEVICE_DNA, 8, dna);
+
+  sbp_send_msg(SBP_MSG_NAP_DEVICE_DNA_DEVICE, 8, dna);
 }
 
 /** Setup NAP callbacks. */
 void nap_callbacks_setup(void)
 {
   static sbp_msg_callbacks_node_t nap_dna_node;
+  static sbp_msg_callbacks_node_t nap_dna_device_node;
 
-  sbp_register_cbk(SBP_MSG_NAP_DEVICE_DNA, &nap_rd_dna_callback,
+  sbp_register_cbk(SBP_MSG_NAP_DEVICE_DNA_HOST, &nap_rd_dna_callback,
                    &nap_dna_node);
+  /* TODO: This is deprecated. Replaced by above SBP_MSG_NAP_DEVICE_DNA_HOST. */
+  sbp_register_cbk(SBP_MSG_NAP_DEVICE_DNA_DEVICE, &nap_rd_dna_callback,
+                   &nap_dna_device_node);
```

Comment:
I added support for both - but if want to have the firmware to just do the new messages, that's fine to. Should figure out how to support that in piksi_tools decently (--legacy parameter?)